Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Aquality.Selenium.Template.NUnit.csproj

```diff
@@ -1,27 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-
+	<TargetFramework>net6.0</TargetFramework>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
-
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\JavaScripts\GetFullPageHeight.js">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
discussed, should be "do not copy", there is an example how to use it in Aquality.Selenium.Tests project FileDownloadingTests 

    var height = browser.ExecuteScriptFromFile("Resources.JavaScripts.GetFullPageHeight.js");

Reply:
The user data and js scripts type were changed to embedded resources.